clients/python/girder_client/cli.py

```diff
@@ -113,6 +114,15 @@ def format_options(self, ctx, formatter):
         self.format_commands(ctx, formatter)
 
 
+def _parseConfigFile(path):
```

Comment:
Our existing configuration is ini-style files so we should probably remain consistent with that.

Reply:
Should this be added to an existing config file somewhere or pass in the path of a cli specific config file?

Reply:
@zachmullen can chime in here, but I think we should probably copy the configuration path precendence set here: https://github.com/girder/girder/blob/61aa5833e91b43d1ea03fa9506175089/girder/utility/config.py#L43-L50 (with a differently named configuration file).

Reply:
👍  

1. Value of `--config` option if passed.
2. Value of `GIRDER_CLIENT_CONFIG` env var if set.
3. ~/.girder/girder_client.cfg
4. /etc/girder_client.cfg